src/constants/tokenSafety.tsx

```diff
@@ -14,17 +14,36 @@ export function getWarningCopy(warning: Warning | null, plural = false) {
   let heading = null,
     description = null
   if (warning) {
-    if (warning.canProceed) {
-      heading = <Plural value={plural ? 2 : 1} _1="This token isn't verified." other="These tokens aren't verified." />
-      description = <Trans>Please do your own research before trading.</Trans>
-    } else {
-      description = (
-        <Plural
-          value={plural ? 2 : 1}
-          _1="You can't trade this token using the Uniswap App."
-          other="You can't trade these tokens using the Uniswap App."
-        />
-      )
+    switch (warning.level) {
+      case WARNING_LEVEL.MEDIUM:
+        heading = (
+          <Plural
```

Comment:
if you have `plural`, why do you need to pass it into this component? can't you just do `plural ? text1 : text2?`

Reply:
Plural is a replacement for using ternary \<Trans> logic

Reply:
ah